src/databricks/labs/ucx/assessment/azure.py

```diff
@@ -8,17 +8,14 @@
 from databricks.sdk.errors import NotFound
 from databricks.sdk.service.compute import ClusterSource, Policy
 
-from databricks.labs.ucx.assessment.crawlers import (
-    CLIENT_ENDPOINT_LENGTH,
-    SECRET_LIST_LENGTH,
-    SECRET_PATTERN,
-    STORAGE_ACCOUNT_EXTRACT_PATTERN,
-    azure_sp_conf_present_check,
-    logger,
-)
+from databricks.labs.ucx.assessment.crawlers import azure_sp_conf_present_check, logger
 from databricks.labs.ucx.assessment.jobs import JobsMixin
 from databricks.labs.ucx.framework.crawlers import CrawlerBase, SqlBackend
 
+SECRET_PATTERN = r"{{secrets\/(.*)\/(.*)}}"
+TENANT_PATTERN = r"https:\/\/login.microsoftonline.com\/(.*)\/oauth2\/token"
```

Comment:
Technically, this only works for public cloud, but not for govcloud or china

Reply:
expanded regex for govcloud & cn